### DISCUSSION TITLE
`首页过滤` 设置项仅在首页时出现在 `附加功能` 中

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1481] [功能建议 / Ideas]
`首页过滤` 设置项仅在首页的 `附加功能` 中出现

**Comment by the1812** [comment]
自定义顶栏设置项也仅出现在有自定义顶栏的地方(

**Reply by the1812** [reply]
不过如果要严格匹配首页的话, 稍后再看就隐藏不了分区了

**Reply by Geeyun-JY3** [reply]
> 不过如果要严格匹配首页的话, 稍后再看就隐藏不了分区了

除了稍后再看页的分区，还有别的地方需要隐藏 `首页过滤` 功能可以隐藏的元素吗🌚

**Reply by the1812** [reply]
各种分区页面啊

**Reply by Geeyun-JY3** [reply]
> 各种分区页面啊

好像也是哦🌚

**Comment by Geeyun-JY3** [comment]
还有一个事，`隐藏顶部横幅` 其实是不是也可以合并进 `首页过滤` 里

**Reply by Geeyun-JY3** [reply]
`隐藏顶部横幅`也会隐藏子分区首页的横幅，还是分开吧。
